src/sonic-eventd/tools/events_monit_test.py

```diff
@@ -0,0 +1,226 @@
+#! /usr/bin/env python3
+
+from inspect import getframeinfo, stack
+from swsscommon.swsscommon import events_init_publisher, event_publish, FieldValueMap
+from swsscommon.swsscommon import event_receive_op_t, event_receive, events_init_subscriber
+from swsscommon.swsscommon import events_deinit_subscriber, events_deinit_publisher
+import argparse
+import os
+import threading
+import time
+import syslog
+import uuid
+
+chk_log_level = syslog.LOG_ERR
+
+test_source = "sonic-host"
+test_event_tag = "device-test-event"
+test_event_key = "{}:{}".format(test_source, test_event_tag)
+test_event_params = {
+    "sender": os.path.basename(__file__),
+    "reason": "monit periodic test",
+    "batch-id": str(uuid.uuid1()),
+    "index": "0"
+}
+
+# Async connection wait time in milliseconds.
+ASYNC_CONN_WAIT = 300
```

Comment:
LGTM warning: Change to 0.3 or 300/1000.0
This code will work fine in python3, but LGTM warning is for Python2. 
In any case, this fix would be good for any version.